src/main/java/pl/futurecollars/invoicing/controller/taxes/TaxCalculatorController.java

```diff
@@ -0,0 +1,33 @@
+package pl.futurecollars.invoicing.controller.taxes;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.futurecollars.invoicing.service.TaxCalculatorService;
+
+@RequestMapping("/tax")
+@RestController
+@Api(tags = {"Tax-Controller"})
```

Comment:
tutaj wyciągnął bym tylko te adnotacje do oddzielnego interfejsu tak jak masz w InvoiceApi - w przypadku TaxCalculatora nie mamy dużo metod, więc to tak nie przeszkadza, ale fajnie jest zachować spójność i trzymać się już jednej konwencji przy wszystkich kontrolerach